docs/_docs/quickstart.md

```diff
@@ -7,19 +7,24 @@ permalink: /docs/quickstart/
 If you already have [Ruby](https://www.ruby-lang.org/en/downloads/) and [RubyGems](https://rubygems.org/pages/download) installed (see Jekyll's [requirements](/docs/installation/#requirements/)), you can create a new Jekyll site by doing the following:
 
 ```sh
+# Install Jekyll and Bundler through RubyGems
 ~ $ gem install jekyll bundler 
-# installs Jekyll and Bundler through RubyGems
+
+# Create a new Jekyll site at myblog
 ~ $ jekyll new myblog 
-# Creates a new Jekyll site called myblog
+
+# Change into new directory
 ~ $ cd myblog
+
+# Build the site on the preview server
 ~/myblog $ bundle exec jekyll serve 
-# Builds the site on the preview server
+
 # Now browse to http://localhost:4000
 ```
 
-The `jekyll new myblog` command installs a new Jekyll site into a subdirectory called `myblog`. The `jekyll new` command automatically initiates `bundle install` to install the dependencies required. (To skip `bundle install`, you can add the following flag: `jekyll new myblog --skip-bundle`.) {% comment %} why would someone want to skip bundle install -- seems unnecessary to even mention it. {% endcomment %}
+The `jekyll new <PATH>` command installs a new Jekyll site at the path specified &mdash; in this case, in a directory called `myblog`. To install jekyll into the directory you're currently in, run `jekyll new .` (If the existing directory isn't empty, you'll also have to pass the `--force` option with `jekyll new . --force`.)  
 
-To install jekyll into the directory you're currently in, run `jekyll new .` (If the existing directory isn't empty, you'll also have to pass the `--force` option with `jekyll new . --force`.)  
+The `jekyll new` command automatically initiates `bundle install` to install the dependencies required. (If you don't want Bundler to install the gems, add the following option: `jekyll new myblog --skip-bundle`.) 
 
 ## Understanding gem-based themes
 
```

Comment:
good idea. i'm working on integrating this info int themes. i will have an update shortly and then push.